### PR TITLE
IDEMPIERE-5089 Add MSysConfig.ALLOW_REVERSAL_OF_RECONCILED_PAYMENT

### PR DESCRIPTION
**Pull request by hengsin**
https://idempiere.atlassian.net/browse/IDEMPIERE-5089

### CODE REVIEW

File: org.idempiere.test/src/org/idempiere/test/model/MProductTest.java [hunk 1]
```diff
@@ -231,8 +231,9 @@ public void testRemoveStorageRecords() {
 		product.setIsActive(false);
 		assertThrows(AdempiereException.class, () -> product.saveEx());
 		
-		//clear on hand so that we can deactivate product
+		//clear on hand and reservation so that we can deactivate product
```
Comment (CarlosRuiz-globalqss) [inline org.idempiere.test/src/org/idempiere/test/model/MProductTest.java:234]: maybe this change is from another ticket?

File: org.adempiere.base/src/org/compiere/wf/MWFActivity.java [hunk 2]
```diff
@@ -990,7 +990,11 @@ else if (savepoint != null)
 				if (m_process != null)
 				{
 					m_process.setProcessMsg(this.getTextMsg());
-					m_process.saveEx();
+					try {
```
Comment (CarlosRuiz-globalqss) [inline org.adempiere.base/src/org/compiere/wf/MWFActivity.java:993]: this looks dangerous - it is ignoring exceptions when saving MWFProcess - is that what you want?

File: org.idempiere.test/src/org/idempiere/test/model/MProductTest.java (hunk 1 above)
Comment (hengsin) [reply org.idempiere.test/src/org/idempiere/test/model/MProductTest.java:234]: I just fix the unit test error when I'm testing this ticket. I can move it else where if that's better.

Comment (CarlosRuiz-globalqss) [reply org.idempiere.test/src/org/idempiere/test/model/MProductTest.java:234]: no problem - it was just curious

**Review by CarlosRuiz-globalqss — APPROVED** [review]
tested

**Pull request merged**